### PR TITLE
Added pgadmin app

### PR DESCRIPTION
**Pull request by jc21**
## Change Description

Added pgadmin web ui app

### CODE REVIEW

File: app/docker/docker-compose.pgadmin.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      description: 'Web based postgresql administrator`admin@example.com`admin'
+      com.centurylinklabs.watchtower.enable: 'true'
+    restart: on-failure
+    ports:
+      - "${DAB_APPS_PGADMIN_PORT:-5555}:80"
+    env_file:
+      - /tmp/denvmux/pgadmin.env
+    environment:
+      PGADMIN_DEFAULT_EMAIL: "${DAB_APPS_PGADMIN_EMAIL:-admin@example.com}"
+      PGADMIN_DEFAULT_PASSWORD: "${DAB_APPS_PGADMIN_PASSWORD:-admin}"
```
Comment (tommymcguiver) [inline app/docker/docker-compose.pgadmin.yml]: Unsure if it's wise to set a default password, i guess this is consistent with the mssql app ¯\_(ツ)_/¯

File: app/docker/docker-compose.pgadmin.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    volumes:
+      - pgadmin:/var/lib/pgadmin
+    tmpfs:
```
Comment (tommymcguiver) [inline app/docker/docker-compose.pgadmin.yml]: Is tmpfs needed here?

File: app/docker/docker-compose.pgadmin.yml (hunk 1 above)
Comment (jc21) [reply app/docker/docker-compose.pgadmin.yml]: The docker image mentions that a default email and password are required - and they can be changed from within the app when it's up and running i believe

File: app/docker/docker-compose.pgadmin.yml (hunk 2 above)
Comment (jc21) [reply app/docker/docker-compose.pgadmin.yml]: Probably not ¯_(ツ)_/¯

Comment (Nekroze) [reply app/docker/docker-compose.pgadmin.yml]: Probably not necessary no, generally keeps things fast though if it works.

File: app/docker/docker-compose.pgadmin.yml (hunk 1 above)
Comment (Nekroze) [reply app/docker/docker-compose.pgadmin.yml]: The mssql one also said it was required sadly so I guess its fine, it occurs to me that the description does not use string interpolation so `dab apps list` output will be wrong in its username and password columns if you override these defaults.

Comment (jc21) [reply app/docker/docker-compose.pgadmin.yml]: Fixed

File: app/docker/docker-compose.pgadmin.yml (hunk 2 above)
Comment (jc21) [reply app/docker/docker-compose.pgadmin.yml]: Removed

**Comment by tommymcguiver** [comment]
Code LGTM

**Comment by Nekroze** [comment]
Needs a rebase before it can be merged

**Comment by Nekroze** [comment]
Needed a rebase not a merge as the merge commit will show up in the `dab changelog`. You can remove the merge commit from this branch by doing the correct rebase command `git rebase origin/master` where origin is the [upstream](https://github.com/Nekroze/dab) remote, this should complete with no conflicts and the `git log` will no longer show that merge commit.

**Comment by tommymcguiver** [comment]
Issue has become stale, code LGTM. Looks like issue has been addressed.

**Pull request merged**